Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/CustomPaywall.swift

```diff
@@ -36,7 +36,7 @@ struct CustomPaywall: View {
                            condensed: self.condensed,
                            fonts: DefaultPaywallFontProvider(),
                            introEligibility: self.introEligibility ?? .default(),
-                           purchaseHandler: self.purchaseHandler ?? .default()
+                           purchaseHandler: self.purchaseHandler ?? .default(performPurchase: nil, performRestore: nil)
```

Comment:
NIt: Left this in another comment but can we make `.default()` with default `nil` for these arguments?